### PR TITLE
add better error handling for contact form

### LINKED ISSUE #36: contact form never says if message sent or failed

### PR DESCRIPTION
**Pull request by butlerx**
close #36

### CODE REVIEW

File: _config.yml [hunk 1]
```diff
@@ -110,7 +110,7 @@ plugins:
     google_analytics: UA-85359173-1 # enter the tracking ID for your Google Analytics
     google_site_verification: 2WCaCH2N-gzzIIiMIq9ERmtz7cprYKIIPKHoBjzPQYk # enter Google site verification code
     baidu_analytics: # enter Baidu Analytics hash key
-    google_recaptcha: 6LdCuwgUAAAAAKB3yf7AcOpyHxIFKJNaH3cNhmEC
+    google_recaptcha: 6LfClyATAAAAAErMssL8Y2iCwGx2JfdcxQHD8biq
```
Comment (SeanHealy33) [inline _config.yml:113]: should these be here?

File: source/js/contact.js [hunk 2]
```diff
@@ -15,9 +15,15 @@ $(document).ready(function(){
...
-        if(data === "sent") {
-          $("#message").empty().html("Email has been sent");
+      $.get(site + "/api/contact", { email:email, to:to, name:name, subject:subject, text:text, recaptcha:recaptcha}, function(data) {
+        if(!data.formSubmit) {
+          $("#message").empty().html('Error in recaptcha');
+        } else {
+          if (data.error) {
+            $("#message").empty().html(data.error);
+          } else {
+            $("#message").empty().html('Email Sent');
```
Comment (SeanHealy33) [inline source/js/contact.js]: You could simplify with a ternary 

``` javascript
!data.formSubmit ? $("#message").empty().html(data.error) : $("#message").empty().html('Email Sent');`
```
That's a bit wordy so maybe something like 

``` javascript
var message = data.error ? data.error : 'Email Sent';
$("#message").empty().html(message);

```

File: _config.yml (hunk 1 above)
Comment (butlerx) [reply _config.yml:113]: they are site keys so they are embed in the generated static files sent to the user

File: source/js/contact.js (hunk 2 above)
Comment (butlerx) [reply source/js/contact.js]: updated to this

**Review by SeanHealy33 — APPROVED** [review]
